### PR TITLE
[POT] Skip per channel quantizers allining before concats

### PR DESCRIPTION
**Pull request by daniil-lyakhov**
### Details:
 - Bridge nodes for unification now checks are any following quantizable node is producing per channel quantizers. This is needed in `concat` -> `depthwise_conv` pattern on CPU

### Tickets:
 - 86063

### CODE REVIEW

File: tools/pot/openvino/tools/pot/algorithms/quantization/fake_quantize_configuration.py [hunk 1]
```diff
@@ -290,6 +290,15 @@ def _get_unified_scales_ops(hw_ops_):
                 unified_scales_ops_.append(hw_op)
         return unified_scales_ops_
 
+    def _get_quantizable_per_ch_ops(hw_cfg_):
```
Comment (AlexKoff88) [inline tools/pot/openvino/tools/pot/algorithms/quantization/fake_quantize_configuration.py:293]: A graph test for the use case is required here.

Comment (daniil-lyakhov) [reply tools/pot/openvino/tools/pot/algorithms/quantization/fake_quantize_configuration.py:293]: I've added test into `test_unify_scales.py`. The graph tests check only FQ insertion and doesn't call FQ unification algo. I'll discuss test coverage with @KodiaqQ

File: tools/pot/tests/test_graph_with_stats.py [hunk 2]
```diff
@@ -17,7 +17,8 @@
 TEST_MODELS = [
     ('resnet_example', 'pytorch', 'DefaultQuantization', 'CPU'),
     ('resnet_example', 'pytorch', 'RangeSupervision', 'ANY'),
-    ('act_act_example', 'pytorch', 'DefaultQuantization', 'GNA')
+    ('act_act_example', 'pytorch', 'DefaultQuantization', 'GNA'),
+    ('concat_depthwise_model', 'pytorch', 'MinMaxQuantization', 'CPU'),
```
Comment (daniil-lyakhov) [inline tools/pot/tests/test_graph_with_stats.py:21]: xml ref file contains fq shapes, so it's checking that FQ before concat are perchannel

Comment (KodiaqQ) [reply tools/pot/tests/test_graph_with_stats.py:21]: Good! So, adding in the test_unify_scales.py is not needed anymore?

Comment (daniil-lyakhov) [reply tools/pot/tests/test_graph_with_stats.py:21]: `test_graph_with_stats` is calling `find_unify_scales` in
1) `get_configuration_by_preset`, where unified quantizers affects fq configuration, which is used for activation statistics layout computation
2) `compute_statistics_layouts`, where unified quantizers affects fq configuration, which is used for filling fq parammeters
3) `unify_fq_scales` to actually unify computed parameters of FQs

And in `test_unify_scales` modified function `find_unify_scales` is called explicitly, and output values is compared with reference values explicitly. This test is more like an unit test, and I suggest to keep it as well. It shouldn't increase testing time in any significant extent

**Review by AlexKoff88 — APPROVED** [review]

**Comment by daniil-lyakhov** [comment]
@AlexKoff88, please merge it if you don't have any comments

**Review by KodiaqQ — APPROVED** [review]

**Pull request merged**